# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2023-12-13 17:33+0330\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n > 1);\n"

#: analyze/middleware/liscensing_middleware.py:25
msgid "liscence has been expired"
msgstr "لایسنس شما به اتمام رسیده است"

#: analyze/views.py:81
msgid "user daas acoount has been block due to many file change attempts"
msgstr "حساب شما به دلیل تعداد تغییر فایل مسدود شده است"

#: analyze/views.py:83 analyze/views.py:85
msgid "server down, please try again later..."
msgstr "خطایی رخ داده است"

#: analyze/views.py:169 analyze/views.py:175 analyze/views.py:183
msgid "you are not permitted to download this type of file"
msgstr "شما اجازه دانلود این فایل را ندارید"

#: analyze/views.py:188
msgid "you are not permitted to upload this type of file"
msgstr "شما اجازه اپلود این فایل را ندارید"

#: analyze/views.py:190
msgid "scanning files in process"
msgstr "در حال اسکن فایل"

#: analyze/views.py:194
msgid "file not sent properly"
msgstr "فایل درست فرستاده نشده است"

#: analyze/views.py:197
msgid "something's wrong"
msgstr "خطایی رخ داده است"

#: analyze/views.py:244
msgid "server down, please try later..."
msgstr "خطایی رخ داده است"

#: analyze/views.py:299
msgid "new mimetype, extension added to database successfully"
msgstr "مایم تایپ و اکستنشن جدید ذخیره شد"

#: analyze/views.py:302
msgid "something's wrong, try later"
msgstr "خطایی رخ داده است"

#: malware_analysis/settings.py:143
msgid "Farsi"
msgstr ""

#: malware_analysis/settings.py:144
msgid "English"
msgstr ""

#: utils/functions.py:30 utils/functions.py:34
msgid "invalid data passed"
msgstr "مقادیر اشتباه است"

#: utils/functions.py:32 utils/functions.py:36
msgid "server down, try later"
msgstr "خطایی رخ داده است"

#~ msgid ""
#~ "content of the file does not match it's extension, contact your "
#~ "administrator"
#~ msgstr "کانتنت فایل و اکستنشن آن مطابقت ندارد. با ادمین تماس بگیرید"
